syntax="proto3";
package IM;

import "chat.proto";

message MessageCach
{
    int64 user_id			= 1; // 用户id
    repeated ChatPkt chat_message	= 2; // 离线消息集合
}
